daemons/attrd/attrd_corosync.c

```diff
@@ -316,7 +317,7 @@ update_attr_on_host(attribute_t *a, crm_node_t *peer, xmlNode *xml, const char *
 }
 
 static void
-attrd_peer_update_one(crm_node_t *peer, xmlNode *xml, bool filter)
+attrd_peer_update_one(const crm_node_t *peer, xmlNode *xml, bool filter)
```

Comment:
`xml` can be const

Reply:
There's an xml_remove_prop() in this one